code/modules/mob/living/silicon/robot/robot_damage.dm

```diff
@@ -4,8 +4,15 @@
 		stat = CONSCIOUS
 		return
 	health = maxHealth - (getBruteLoss() + getFireLoss())

```

Comment:
@Crazylemon64 I made that other updateHealth proc- but before I made it, that health setting was directly in Life(), yet this has always existed. I don't understand why.